### PR TITLE
[LDR R2] docs update

### PR DESCRIPTION
**Pull request by t7ko-seagate**

### CODE REVIEW

**Review by AjinkyaDhumal — APPROVED** [review]

**Review by amitwac2608 — COMMENTED** [review]
Ask @shalakadharap  also to review these. I guess some of the ldap configuration and replication configuration is going to change.

File: docs/LB_and_HA/keepalived_config.md [hunk 1]
```diff
@@ -47,20 +47,20 @@ times more requests than the third node.
...
-accross several nodes.
+across several nodes.
 
 Even load distribution in case of failover requires N * (N - 1) IP addresses
 for the cluster of N nodes.
 
 There could be different configurations as well. For example
 4 IP addresses per node for cluster of N nodes - in case of failover
-this configuratons leads to 25% load increase for some nodes.
+this configurations leads to 25% load increase for some nodes.
```
[the remark this answers is not in the document]
Comment (t7ko-seagate) [reply docs/LB_and_HA/keepalived_config.md]: Done.

File: docs/LB_and_HA/keepalived_config.md [hunk 2]
```diff
@@ -47,20 +47,20 @@ times more requests than the third node.
...
 5 IP addresses per node for cluster of N nodes - in case of failover
-this configuratons leads to 20% load increase for some nodes.
+this configurations leads to 20% load increase for some nodes.
```
[the remark this answers is not in the document]
Comment (t7ko-seagate) [reply docs/LB_and_HA/keepalived_config.md]: Fixed.

File: docs/LB_and_HA/keepalived_config.md [hunk 3]
```diff
@@ -148,6 +148,11 @@ same *vrrp_instance*. The order *backup* nodes failover ip addresses is determin
 the *priority* field - next active node is the node with the highest *priority*. Each
 *vrrp_instance* must have unique *virtual_router_id* in scope of a single sub-network.
 
+### Limitations
+
+*virtual_router_id* is a single byte value from the range 1 to 255, so only 255
```
[the remark this answers is not in the document]
Comment (t7ko-seagate) [reply docs/LB_and_HA/keepalived_config.md]: Fixed.

**Review by shalakadharap — APPROVED** [review]

**Review by amitwac2608 — APPROVED** [review]

File: docs/R2-setup/3 Node Manual Hare Cluster Setup.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+For any questions about this software or licensing,
+please email opensource@seagate.com or cortx-questions@seagate.com.
+
+## Steps to configure 3 node cluster manually
+
+1. **Order 3 fresh ssc vms from** -- https://ssc-cloud.colo.seagate.com/ui/service/catalogs
```
[the remark this answers is not in the document]
Comment (t7ko-seagate) [reply docs/R2-setup/3 Node Manual Hare Cluster Setup.md:24]: Decided to merge as is, and update later when necessary.

File: docs/R2-setup/3 Node Manual Hare Cluster Setup.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+2. **Perform manual HARE / Motr cluster setup steps mentioned at below location** - 
+
+   https://seagatetechnology.sharepoint.com/:w:/s/gteamdrv1/tdrive1224/EfvqJjlha8pNkOeIfCgDywUBn0YNIYGT6-tWMREx9iGxpw?e=D9OLR8
```
[the remark this answers is not in the document]
Comment (t7ko-seagate) [reply docs/R2-setup/3 Node Manual Hare Cluster Setup.md:28]: Discussed with team offline and decided to merge as is, and update later when necessary.

**Pull request merged**